CHANGELOG.md

```diff
@@ -8,6 +8,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 - Add functional test coverage for Network Explorer metrics ([#684](https://github.com/signalfx/splunk-otel-collector-chart/pull/684)
 - Apply the same resources to init containers as allocated to the otel agent container ([#690](https://github.com/signalfx/splunk-otel-collector-chart/pull/690)
+- Add basic support for deploying OpenTelemetry Operator as a subchart [#691](https://github.com/signalfx/splunk-otel-collector-chart/pull/691)
+  - Disabled for all users until the feature is more mature
```

Comment:
We will not be enabling it by default anyway, right?

Reply:
Let's maybe remove this line then? It sounds like it'll be enabled at some point